cfgov/v1/models/learn_page.py

```diff
@@ -40,7 +40,7 @@ class AbstractFilterPage(CFGOVPage):
     # General content tab panels
     content_panels = CFGOVPage.content_panels + [
         StreamFieldPanel('header'),
-        StreamFieldPanel('content'),
+        # StreamFieldPanel('content'),
```

Comment:
Temporary - need to figure out why we get this error when it's not commented out: `django.core.exceptions.FieldDoesNotExist: AbstractFilterPage has no field named 'content' `